### DISCUSSION TITLE
🧩 Tai 浏览器插件下载

### DISCUSSION THREAD

**Discussion by noberumotto** [Planshit/Tai#279] [Announcements]
#### 要使用网页浏览数据统计必须安装相应的浏览器插件

目前只完成了 Chrome 浏览器拓展的开发，后续更新了其他浏览器的都会发布到这个帖子中。也欢迎大家参与其他浏览器的Tai拓展开发，参考源码：[WebExtensions/Chrome/service-worker.js](https://github.com/Planshit/Tai/blob/1.5.0.3/WebExtensions/Chrome/service-worker.js)

> Edge浏览器与Chrome可以共用同一个拓展

#### 从拓展商店
1.  [Chrome 浏览器 Tai 拓展](https://chromewebstore.google.com/detail/tai-sentry/fmjgafoilnpanbpgdkljkjfjgainboim)

#### 手动安装
1. 在下方下载相应的浏览器拓展压缩包，解压到任意位置；
2. 打开浏览器，打开拓展管理，启用 `开发者模式` ，点击 `加载已解压的拓展程序` ，选择第1步解压的文件夹完成拓展安装。

### 使用

在 Tai 的设置 > 常规 > 功能 中启用网站浏览统计功能，当此功能正常运行时，Tai 浏览器拓展的小图标会呈彩色（表示已与Tai正常连接），如果关闭了Tai或者出现问题无法连接时图标会变为灰色，此时无法统计浏览器的访问数据。

### 下载

手动安装以支持 Edge 浏览器，下载压缩包：[Tai Chrome / Edge 浏览器拓展.zip](https://github.com/Planshit/Tai/files/13178024/Tai.Chrome.zip)

**Comment by PMisRIGHT** [comment]
## 能用的FireFox插件

此扩展由[Tai的Chrome扩展](https://github.com/Planshit/Tai/tree/master/WebExtensions/Chrome)重新打包，对应版本为v0.1，只编辑了manifest.json，未对主要功能模块service-worker.js做修改，使用了[Mozilla Polyfill](https://github.com/mozilla/webextension-polyfill/)来避免不兼容的情况。

### 安装

点这个→[下载[c443b65336b74e5090c4-0.1.xpi.zip]](https://github.com/Planshit/Tai/files/14470048/c443b65336b74e5090c4-0.1.xpi.zip)


下载好后，打开FireFox的扩展管理页面，在右上角小齿轮中选择“从文件安装附加组件”，右上角点击蓝色“添加”
![图片](https://github.com/Planshit/Tai/assets/45200086/3f2be0f7-1d65-4a20-94dc-488c2637091f)
装好了打开扩展菜单看看图标是 **彩色** 的就说明成功连接到Tai了（如下），别忘了在Tai的设置里打开网站浏览统计
![图片](https://github.com/Planshit/Tai/assets/45200086/4373cf46-1fc8-4a3c-b014-75b1089ed071)

### 使用

在 Tai 的设置 > 常规 > 功能 中启用网站浏览统计功能，当此功能正常运行时，Tai 浏览器拓展的小图标会呈彩色（表示已与Tai正常连接），如果关闭了Tai或者出现问题无法连接时图标会变为灰色，此时无法统计浏览器的访问数据。

###  bug反馈

还有详细说明请到https://github.com/Planshit/Tai/discussions/318

**Comment by MC-dusk** [comment]
问个奇怪的问题，用了半年了，为什么我没安装扩展也能统计网页，以至于我一直以为这个功能不需要浏览器扩展
在这个帖子发布（Oct 26, 2023）之前和之后都能统计到啊

**Reply by MC-dusk** [reply]
破案了，用的刚好是需要额外扩展之前的五月发布的老版本，更新就正常了

**Comment by lyy33441314** [comment]
你好这个插件是不是 会影响页面正常的打开速度  打开插件 进不去网页 关闭就立马进入了

**Comment by wangliuzhe** [comment]
报错：WebSocket connection to 'ws://127.0.0.1:8908/TaiWebSentry' failed: Error in connection establishment: net::ERR_CONNECTION_REFUSED
是不是服务器到期了？

**Reply by afterow** [reply]
这个是本地的端口 请你检查浏览器是否能访问这个端口 和有没有开启这个端口

**Reply by ryuujo1573** [reply]
你如果是计算机专业的学生，我建议你早一些另寻出路吧。。。

**Comment by OREO-0214** [comment]
你好，有没有edge浏览器的插件呀，谢谢大佬

**Reply by remnant-song** [reply]
edge和Chrome插件通用的

**Comment by lyy33441314** [comment]
有的 他里面谷歌的其实就可以


 

 	----------回复的邮件信息----------
 	OREO-0214 ***@***.***&gt; 在 2025年11月26日 17:17写道：
      		

 
你好，有没有edge浏览器的插件呀，谢谢大佬

**Reply by OREO-0214** [reply]
你好，没明白在哪里看，我没有chrome，是要下载chrome并导入这个插件后，就可以在edge也使用了嘛。浏览器插件自主导入没有识别到这个插件
![Uploading 微信图片_20251126172702_102_44.png…]()

**Reply by OREO-0214** [reply]
我会啦，直接点选择文件夹就可以了，没注意到以为点其中的文件，谢谢佬！

**Comment by goodfish-jhy** [comment]
插件已上架 [链接](https://addons.mozilla.org/zh-CN/firefox/addon/tai-sentry/)

**Comment by nicolasniki** [comment]
好用，感谢！
